src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -70,7 +70,6 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>SqmOleCommandTarget.tt</DependentUpon>
     </Compile>
-    <Compile Include="Telemetry\SonarLintIntegrationPackage.IOleCommandTarget.cs" />
```

Comment:
There's an assert, but in more general I have a service that I use instead and the responsibility for that initialization needs to be on the service.